.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules/
 package-lock.json
 yarn.lock
 pnpm-lock.yaml
+.history
```

Comment:
why is this here?

Reply:
Using some vscode extensions, such as [local-history](https://github.com/zabel-xyz/local-history) will generate history files.

Reply:
Put it in the local settings for your environment, it does not apply to the project.

Reply:
@Scrum Hello, it has been removed.